app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="row inset">
+<div class="row inset" style="margin-top:60px">
```

Comment:
ditto

Reply:
done, added style to layout.scss and updated file